clients/gateway_test.go

```diff
@@ -26,14 +26,18 @@ func TestStateUpdateUnmarshal(t *testing.T) {
         }
       ]
     },
-    "nonces": {},
+    "nonces": { 
+		"0x37" : "0x44"
+	},
     "deployed_contracts": [
       {
         "address": "0x20cfa74ee3564b4cd5435cdace0f9c4d43b939620e4a0bb5076105df0a626c6",
         "class_hash": "0x10455c752b86932ce552f2b0fe81a880746649b9aee7e0d842bf3f52378f9f8"
       }
 	],
-    "declared_contracts": []
+    "declared_contracts": [
+		"0x3744"
+	]
   }
 }`)
```

Comment:
Can we have a reference for where the json data is gotten from? Adding the feeder gateway url, for instance, so that it'll be clear why these values were updated.

Reply:
I think this should be done for all the `jsonData`.